src/AppChooser/Dialog.vala

```diff
@@ -110,7 +110,8 @@ public class AppChooser.Dialog : Hdy.Window {
         placeholder.show_all ();
 
         listbox = new Gtk.ListBox () {
-            expand = true
+            expand = true,
```

Comment:
```suggestion
            hexpand = true,
            vexpand = true
```

While here, let's do this in preparation to Gtk4.

Reply:
Fixed!